newIDE/app/src/AssetStore/BehaviorStore/BehaviorListItem.js

```diff
@@ -69,10 +69,13 @@ export const BehaviorListItem = ({
 
   // Report the height of the item once it's known.
   const containerRef = React.useRef<?HTMLDivElement>(null);
-  React.useLayoutEffect(() => {
-    if (containerRef.current)
-      onHeightComputed(containerRef.current.getBoundingClientRect().height);
-  });
+  React.useLayoutEffect(
+    () => {
+      if (containerRef.current)
+        onHeightComputed(containerRef.current.getBoundingClientRect().height);
```

Comment:
This Layout effect was the one causing errors. sometimes we see an infinite loop of updates happening on this list.
Is it possible that it keeps getting called?
I've added the dependency array as a way to fix that, but unsure `onHeightComputed` is stable enough...

Reply:
Can we check where onHeightComputed is passed?

Reply:
It is a bit convoluted tbh, onHeightComputed is created inside another callback defined in the ListSearchResults, which is called when rendering the row with the BehaviorListItem.
This function recomputes the grid (which is why it can trigger multiple renders)

It does feel safer to call this effect when the function is ready, rather than on every render though.
From my tests, I haven't noticed it not working as expected but maybe I'm missing something